### DISCUSSION TITLE
No Internet Notification in Splash Screen [ MV Fluxstore - WCFM ]

### DISCUSSION THREAD

**Discussion by nikhilbhoyar** [inspireui/support#5374] [Working on Features]
When any user open MV Fluxtstore application
and if there is no Internet Connection.

**NO INTERNET Message Dialog** should show in Splash Screen itself.
Asking User to check internet connection and restricting them to open Application.

This is recommended feature,
As if we are using remote Config URL
and user phone is not connected to internet.
Then Application become White Blank.

**Comment by simonpham** [comment]
Available in FluxStore 3.4.0.
The app will be restricted with a Refresh button.
